lang/en/passwords.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Password Reset Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are the default lines which match reasons
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset' => 'Your password has been reset.',
```

Comment:
```suggestion
    'Your password has been reset.' => 'Your password has been reset.',
```

I mentioned in a previous review that you should use the translation strings as the keys. Do this everywhere.


https://laravel.com/docs/10.x/localization#using-translation-strings-as-keys

Reply:
Search if you can regenerate them in the particular way discussed or if you can copy them from somewhere. 

Finish this so translation work can proceed.